app/services/publish_content_item.rb

```diff
@@ -0,0 +1,34 @@
+class PublishContentItem
```

Comment:
Would this be more consistently implemented by using `PublishingApiWorker` and adding the lookup by base path in a method in https://github.com/alphagov/whitehall/blob/master/lib/whitehall/publishing_api.rb That would avoid a lot of the duplication. It would also get the Sidekiq stuff and automated queuing of sync checks that we recently added that that worker.